<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_find">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲特征：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_find">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>61</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Easy Touch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_song">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_next">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_replay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_volumn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>28</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_find">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>271</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>511</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pb_minimum">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_close">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_onPlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_duration">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>33</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_play">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>播放列表</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
